src/services/syncService.js

```
@@ -0,0 +1,147 @@
+      for (const tile of userData.customTiles) {
+        // Check if tile already exists by matching action and group
+        const existingTiles = await getTiles({
+          group: tile.group,
+          action: tile.action,
+        });
+
+        if (!existingTiles.length) {
+          await addCustomTile(tile);
+        }
+      }
+      console.log(userData.customTiles.length + ' custom tiles imported');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)